pyuvdata/uvbeam.py

```diff
@@ -978,6 +977,43 @@ def __iadd__(self, other):
         self.__add__(other, inplace=True)
         return self
 
+    def get_beam_area(self,stokes='I'):
+        """
+        Computes the integral of the beam, which has units of steradians
+        """
+        if self.beam_type is not 'power':
+            raise ValueError('beam_type must be power')
+        if self._data_normalization.value != 'peak':
+            print self._data_normalization.value
+            raise ValueError('beam must be peak normalized')
```

Comment:
Note that a 'peak' normalization means that each frequency has been separately peak normalized and the intrinsic variation in beam response with frequency has been pushed into the bandpass variable. Is that what you want? Do you want to take the bandpass into account in any way?

Reply:
I think we do want peak normalization. I'll check with some people, but I think it's peak that we want.